cobra/core/DictList.py

```diff
@@ -54,7 +54,7 @@ def list_attr(self, attribute):
         """
         return [getattr(i, attribute) for i in self]
 
-    def query(self, search_function, attribute="id"):
+    def query(self, search_function, attribute=None):
```

Comment:
While already have a quite a few statements like `model.reactions.query('EX')` in my notebooks, they would all break with this modification.. Perhaps a good solutions for both would be to add that if search_functions is a RE, and attribute is None then attribute is 'id'?